src/sentry/integrations/bitbucket_server/repository.py

```diff
@@ -93,16 +95,16 @@ def on_delete_repository(self, repo):
                 return
             raise
 
-    def _format_commits(self, repo, commit_list):
+    def _format_commits(self, client, repo, commit_list):
         return [
             {
                 "id": c["id"],
                 "repository": repo.name,
                 "author_email": c["author"]["emailAddress"],
-                "author_name": c["author"]["displayName"],
+                "author_name": c["author"]["name"],
```

Comment:
@pierredup Would it make sense to use `displayName` if it's available and default to `name` if it's not? It's extra logic but it seems to me that the `displayName` is better to show.